Tests/PackageRegistryTests/SignatureValidationTests.swift

```diff
@@ -1673,15 +1472,7 @@ final class SignatureValidationTests: XCTestCase {
         }
     }
 
-    func testSignedManifest_certificateNotTrusted_shouldPrompt() async throws {
-        let registryURL = URL("https://packages.example.com")
-        let identity = PackageIdentity.plain("mona.LinkedList")
-        let package = identity.registry!
-        let version = Version("1.1.1")
-        let metadataURL = URL("\(registryURL)/\(package.scope)/\(package.name)/\(version)")
-        let checksum = "a2ac54cf25fbc1ad0028f03f0aa4b96833b83bb05a14e510892bb27dea4dc812"
-
-        let keyAndCertChain = try self.ecSelfSignedTestKeyAndCertChain()
+    func testSignedManifest_certificateNotTrusted_shouldPrompt() async throws {        let keyAndCertChain = try self.ecSelfSignedTestKeyAndCertChain()
```

Comment:
nit (blocking): minor formatting issue.
```suggestion
    func testSignedManifest_certificateNotTrusted_shouldPrompt() async throws {
        let keyAndCertChain = try self.ecSelfSignedTestKeyAndCertChain()
```